src/bin/hash_file.rs

```diff
@@ -1,9 +1,9 @@
 extern crate twox_hash;
 
 use std::env;
-use std::hash::Hasher;
 use std::fs::File;
-use std::io::{BufRead,BufReader};
+use std::hash::Hasher;
+use std::io::{BufRead, BufReader};
```

Comment:
While the formatting of this crate doesn’t currently match the rustfmt standard, I’d appreciate it if you left the formatting as-is for code you aren’t adding. Feel free to rustfmt your *new* additions however, as I would like to do some cleanup eventually.

Reply:
I understand, I have removed the format commit.